js/interface.js

```diff
@@ -16,6 +16,7 @@ var $searchType = $('.search-type');
 var $searchTerm = $('.search-term');
 var $fileTable = $('.file-table');
 var $pagination = $('.pagination');
+var $goToFolderAlertTimeout = 5000;
```

Comment:
only use the `$` prefix for jquery elements

Reply:
fixed